demos/storybook/stories/drawer/within-drawer-layout.stories.ts

```diff
@@ -3,32 +3,46 @@ import * as Colors from '@pxblue/colors';
 import { nestedNavGroup } from './with-nested-nav-items.stories';
 import { DrawerNavItem } from '@pxblue/angular-components';
 import { getDirection } from '@pxblue/storybook-rtl-addon';
+const headerImage = require('../../assets/topology_40.png');
+const railImage = require('../../assets/touchscreen_large.svg');
 
 const items = [...nestedNavGroup];
 
 export const withinDrawerLayout = (): any => ({
+    styles: [
+        `
+        :host { 
+            height: 100%; 
+            width: 100%;
+        }
+        ::ng-deep .pxb-drawer-header-background {
```

Comment:
Why do we need ng-deep here? I thought our component encapsulation rules let us avoid this.

Reply:
Storybook style overrides require an ng-deep because each story has the default angular ViewEncapsulation applied to it which would scope each stories' styles to only affect local elements.  

We won't have this problem in the showcase because our demo project has ViewEncapuslation.None.